examples/dynamo/sam/README.md

```diff
@@ -0,0 +1,39 @@
+# SAM2 model compilation using Torch-TensorRT
+
+This example illustrates the state of the art model `Segment Anything Model 2` optimized using
+Torch-TensorRT.
+
+## Dependencies
+Install the following dependencies before running the model.
+
+```py
+pip install -r ../requirements.txt
+```
+
+Certain custom modifications are required to ensure the model is exported successfully. To apply these changes, please install SAM2 using the following fork : https://github.com/chohk88/sam2/tree/torch-trt (<a href="https://github.com/chohk88/sam2/tree/torch-trt?tab=readme-ov-file#installation">Installation</a>)
```

Comment:
Can you describe the modifications?

Reply:
I have added a description of the modifications to the SAMv2 model.